components/lock.rst

```diff
@@ -228,61 +227,64 @@ Lock will be released automatically as soon as one process finishes::
     }
     // ...
 
-To disable this behavior, set to ``false`` the third argument of
-``LockFactory::createLock()``. That will make the lock acquired for 3600 seconds
-or until ``Lock::release()`` is called.
+To disable this behavior, set the ``autoRelease`` argument of
```

Comment:
Before it was named flag, now argument.

Should be the same

Reply:
Tweaked the other sentence to use argument there as well.